docs/content/ingestion/faq.md

```diff
@@ -2,6 +2,10 @@
 layout: doc_page
 ---
 
+## My Data isn't being loaded
+
+If you are trying to stream in historical (not current time) data into Druid and you are using the [serverTime](../ingestion/realtime-ingestion.html) rejection policy in your ingestion spec (the default rejection policy), Druid will not ingest this data as it is outside of the acceptable window period. You can verify this is what is happening by looking at the logs of your real-time process for log lines containing "ingest/events/*". These metrics will indicate the events ingested, rejected, etc. We recommend using batch ingestion methods for historical data or using the [messageTime](../ingestion/realtime-ingestion.html) rejection policy.
```

Comment:
I think language like this is pretty clear for this kind of stuff:

- "[blah] is provided for use in evaluations, but in production, [other blah] should be used instead"
- "[blah] is provided for testing and prototyping, but is not appropriate for production"